src/test-and-compile.bat

```diff
@@ -0,0 +1,45 @@
+setlocal
+@echo off
+
+rem maven plugin version
+set VER=%1
+set BASEDIR=%CD%
+set BIN=%BASEDIR%\bin
+rem destination of the test reports
+set TEST=%BASEDIR%\test-results
+rem failure file
+set FAILED=%TEST%\failed
+set NAME=durable_task_monitor
+
+rem gotestsum will generate junit test reports. v0.4.2 is the latest compatible with golang 1.14
+del /s /q %TEST%
+mkdir %TEST%
+cd %BASEDIR%\pkg\common
+go mod tidy
+rem https://golang.org/issue/36568 prevents downloading gotestsum, unable to generate junit test reports
+rem go get -v gotest.tools/gotestsum@v0.4.2
+rem gotestsum --format standard-verbose --junitfile %TEST%\common-unit-tests.xml
+go test -v
+if NOT %ERRORLEVEL% == 0 echo command>>%FAILED%
+rem TODO test windows
+rem cd %BASEDIR%\cmd\windows
+rem go mod tidy
+rem go get -v gotest.tools/gotestsum@v0.4.2
+rem gotestsum --format standard-verbose --junitfile %TEST%\bash-unit-tests.xml
+rem if NOT %ERRORLEVEL% == 0 echo windows>>%FAILED%
+
+rem build the binaries
+del /s /q %BIN%
+mkdir %BIN%
+cd %BASEDIR%/cmd/bash
+set CGO_ENABLED=0& set GOOS=darwin& set GOARCH=amd64& go build -a -o %BIN%/%NAME%_%VER%_darwin_64
+set CGO_ENABLED=0& set GOOS=darwin& set GOARCH=amd64& go build -a -o %BIN%/%NAME%_%VER%_darwin_32
+set CGO_ENABLED=0& set GOOS=darwin& set GOARCH=amd64& go build -a -o %BIN%/%NAME%_%VER%_unix_64
+set CGO_ENABLED=0& set GOOS=darwin& set GOARCH=amd64& go build -a -o %BIN%/%NAME%_%VER%_unix_32
```

Comment:
Still `GOOS=darwin`? And what sort of platform is `unix` exactly?

Reply:
ugh, you're right this is all sorts of wrong. 😢 These lines were all from when I was first figuring out all the cross-compiling stuff and once i got it to work, I never bothered changing.

Should be `GOOS=linux` and platform `linux`

Reply:
cleaned up poor naming in 48efa4fe70f4ba2b9a29944253e7934a4afd8406 and 48efa4fe70f4ba2b9a29944253e7934a4afd8406